liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/AbstractLiquibaseMojo.java

```diff
@@ -839,19 +841,31 @@ public void configureFieldsAndValues() throws MojoExecutionException, MojoFailur
         if (propertyFile != null) {
             getLog().info("Parsing Liquibase Properties File");
             getLog().info("  File: " + propertyFile);
-            try (InputStream is = handlePropertyFileInputStream(propertyFile)) {
+            try (InputStream _is = handlePropertyFileInputStream(propertyFile)) {
```

Comment:
Similarly to my previous comments, I would suggest extracting this logic as a function of `AbstractLiquibaseMojo` class, similarly as suggested for `CreateProcedure` change.